src/metax_api/api/rest/base/views/dataset_view.py

```diff
@@ -334,6 +334,9 @@ def list_datasets(self, request):
         self.queryset_search_params = {'id__in': ids}
         return super(DatasetViewSet, self).list(request)
 
+    def destroy_bulk(self, request, *args, **kwargs):
+        return self.service_class.destroy_bulk(request.data)
```

Comment:
I think destroy_many would be more descriptive name

Reply:
Used destroy_bulk because it's already introduced in common_view row 309:
`    def destroy_bulk(self, request, *args, **kwargs):
        # not allowed... for now
        return Response({}, status=status.HTTP_405_METHOD_NOT_ALLOWED)`
but actually I am not sure if it's meant for this kind perform.
@EarthModule please elaborate on this if you have any opinion .

Reply:
I think its the same as the other one. destroy_many would have been more descriptive IMHO, but that is just me.

Reply:
destroy_bulk and actually update_bulk are used through out Metax, so I think we'll leave it like that for now.

Reply:
agreed, this was not negative feedback, I just wanted to probe alternatives :)

Reply:
and it's the way to go! :)